modules/ROOT/pages/troubleshoot-saml-assertions-task.adoc

```diff
@@ -24,4 +24,14 @@ https://www.mulesoft.com/support-login
 ----
 
 . Provide a screen shot of your SAML configuration in Anypoint Platform.
-. Include an XML SAML assertion.
\ No newline at end of file
+. Include an XML SAML assertion.
+
+== Example: Chrome Developer Tools SAML Response Element
+
+The following image shows an instance of Chrome developer tools that contains a SAML response element after logging in to Anypoint Platform using SSO.
+
+.SAML response element after logging in to Anypoint Platform in Google Chrome.
+image::iam-chromedevtools.png[]
+
+In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to log in to Anypoint Platform using your SSO.
+The login is successful, and you can expand `receive-id` and access form data that contains your SAML response element (3). Note that unsuccessful login attempts still generate SAML response elements.  
```

Comment:
```suggestion
Because the login is successful, you can expand `receive-id` and access form data that contains your SAML response element (3). Note that unsuccessful login attempts still generate SAML response elements.  
```
So...are these two sentences saying that whether the login is successful or not, you can expand...to see the SAML response? Maybe what we want, if so, is:

```suggestion
In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to log in to Anypoint Platform using your SSO ID. Whether the login is successful or not, you can expand `receive-id` and access form data that contains your SAML response element (3).
```

Reply:
Contextually speaking, the first option is more useful.